about-us.php

```diff
@@ -1,4 +1,41 @@
 <?php include 'includes/header.php';?>
 
+<div class="hero-image full-width bottom-spacer">
```

Comment:
Hero image code should not be in this pull request.

Reply:
done